packages/get-platform/src/getPlatform.ts

```diff
@@ -197,9 +196,9 @@ export async function getSSLVersion(args: GetOpenSSLVersionParams): Promise<GetO
 
   debug('Falling back to "ldconfig" and other generic paths')
   const libsslFilename: string | undefined = await getFirstSuccessfulExec([
-    'ldconfig -p | sed "s/.*=>s*//" | sed "s/.*///" | grep ssl | sort -r',
-    'ls -r /lib64 | grep ssl',
-    'ls -r /usr/lib64 | grep ssl',
+    'ldconfig -p | sed "s/.*=>s*//" | sed "s/.*///" | grep ssl',
```

Comment:
maybe

```suggestion
    'ldconfig -p | sed "s/.*=>s*//" | sed "s/.*///" | grep ssl | sort',
```

to be safe

Reply:
or is it guaranteed to be lexicographically ordered already?

Reply:
Hi, I also had second thoughts on this. It's probably better to keep `| sort` and be sure with it, as the paths returned by `ldconfig -p` may have wildly different basedirs